src/SFML/Graphics/CircleShape.cpp

```diff
@@ -72,11 +72,10 @@ std::size_t CircleShape::getPointCount() const
 ////////////////////////////////////////////////////////////
 Vector2f CircleShape::getPoint(std::size_t index) const
 {
-    static const float pi = 3.141592654f;
-
-    float angle = static_cast<float>(index) * 2.f * pi / static_cast<float>(m_pointCount) - pi / 2.f;
-    float x = std::cos(angle) * m_radius;
-    float y = std::sin(angle) * m_radius;
+    Angle angle = static_cast<float>(index) / static_cast<float>(m_pointCount) * sf::degrees(360) - sf::degrees(180);
```

Comment:
Here, `pi / 2.f` should be `sf::degrees(90)` instead of `sf::degrees(180)`.

Reply:
Thank you! It should be fixed now.